README.md

```diff
@@ -4,227 +4,194 @@
 
-## Running tests
+### Apisonator failed jobs rescheduler
 
-You can either run them manually in the container-based development environment
-or have a container launched just for running the tests. For the latter you just
-need to run `make test`, and you can configure any additional environment
-variables for the test scripts like so:
+This is a simple task that acts as a cron scheduler to requeue jobs that failed
+when being processed by an Apisonator worker. The jobs are requeued into
+the Redis database.
 
-> `$ make DOCKER_OPTS="-e TEST_ALL_RUBIES=1" test`
+This component may also be referred as 'backend-cron'. 
 
-Another alternative to execute all tests from within the development environment
-where you should not need to manually start/stop the services before is to execute
-them from inside the development environment via this script:
+## Development
 
-> `$ script/test`
+See the file [DEVELOPMENT](DEVELOPMENT.md)
 
-### Running tests (advanced)
+## Deployment set up
 
-In order to run the tests manually in the container-based development, the services
-needed to run them correctly must be started before with:
+### Prerequisites
 
-> `$ script/services start`
-
-Then you can execute the following commands to execute all tests:
+* Docker (requires version 1.10.0 or later)
+* A Redis database
 
-> `$ bundle exec rake`
+The first thing you will need is cloning the project:
+> `$ git clone git@github.com:3scale/apisonator.git`
 
-In case you need it/want it, it is possible to manually stop the services by executing:
-> `$ script/services stop`
+Next cd into the directory, `cd apisonator`.
 
-You can also specify execution of individual tests or type of tests.
+### Apisonator image generation
 
-If the test is a Unit::Test type test (located in test directory) you can execute:
-> `$ bundle exec rake test:[unit|integration|special] [TEST=<test-file]>`
+Go to the `openshift` directory and execute `make build`. This will generate
+a local docker image named `amp:apisonator-<version_number>` based on CentOS 7.
 
-If the test is a RSpec type test (located in spec directory) you can execute:
-> `$ bundle exec rake spec:[unit|integration|acceptance|api|use_cases] SPEC=<test-file>`
+### Configuration
 
-Alternatively there is another syntax to execute a RSpec type test:
-> `$ bundle exec rake spec:specific[<test-file>]`
+To run any Apisonator component, application-related environment variables must
+be previously set. This can be done by setting them via the `--env` flag in
+Docker or by placing them in a ENV file and setting the ENV file in Docker via
+the `--env-file` flag.
 
-where in this last case the [ ] characters must be literally placed.
+The most important variables to set are:
 
-Finally, you can also execute tests with rspec directly with:
+ * CONFIG_QUEUES_MASTER_NAME: Set this to <hostname>:<port> of where the Redis
+   has been installed
+ * CONFIG_REDIS_PROXY: Set this to <hostname>:<port> of where the Redis
```

Comment:
This and the other setting one above receive a Redis URL - hostname:port is deprecated and discouraged. We might want to mention support for Sentinels (in which case these two settings change their meaning) but that is complex enough that might be better explained in its own section.